qemu/tests/ctrl_vlan.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_net
+from virttest import utils_test
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run ctrl_vlan check test.
+
+    1) Boot vm with ctrl_vlan=on/off
+    2) Verify if netkvm.sys is enabled in guest(only windows)
+    3) Check vlan table in rx-filter information
+    4) If ctrl_vlan=on, do step 5-6
+    5) Set vlan in guest
+    6) Check vlan table in rx-filter information again
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    def verify_vlan_table(vm, expect_vlan=None):
+        error_context.context("Check vlan table in rx-filter", logging.info)
+        query_cmd = "query-rx-filter name=%s" % vm.virtnet[0].device_id
```

Comment:
Do you need to put query_cmd to cfg?

Reply:
Thanks, I think it is ok to put query_cmd here.